src/Template/Bake/Model/entity.ctp

```diff
@@ -13,6 +13,13 @@
  * @license       http://www.opensource.org/licenses/mit-license.php MIT License
  */
 
+use Cake\ORM\Association;
+
+$propertyHintMap = null;
```

Comment:
@markstory That would result in an empty line after the description in case no properties are being available. Should be an edge case in userland I guess, but tests would need to adapt to this behavior. I can change it if you're OK with that?